backend/src/test/java/com/woowacourse/levellog/domain/InterviewQuestionTest.java

```diff
@@ -0,0 +1,115 @@
+import com.woowacourse.levellog.common.exception.UnauthorizedException;
+import com.woowacourse.levellog.interview_question.domain.InterviewQuestion;
+import com.woowacourse.levellog.levellog.domain.Levellog;
+import com.woowacourse.levellog.member.domain.Member;
+import com.woowacourse.levellog.team.domain.Team;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("InterviewQuestion 클래스의")
+class InterviewQuestionTest {
+
+    @Nested
+    @DisplayName("생성자는")
+    class ConstructorTest {
+
+        @Test
+        @DisplayName("인터뷰 질문을 생성한다.")
+        void success() {
+            // given
+            final Member from = new Member("페퍼", 1111, "pepper.png");
+            final Member to = new Member("이브", 123123, "image.png");
+            final Team team = new Team("잠실 네오조", "트랙룸", LocalDateTime.now().plusDays(3), "jamsil.img");
+            final Levellog levellog = Levellog.of(to, team, "레벨로그 작성 내용");
+            final String content = "스프링이란?";
+
+            // when & then
+            assertDoesNotThrow(() -> InterviewQuestion.of(from, to, levellog, content));
+        }
+
+        @Test
+        @DisplayName("인터뷰 질문의 길이가 255자 초과일 경우 예외를 던진다.")
+        void interviewQuestion_contentLength_exception() {
+            // given
+            final Member from = new Member("페퍼", 1111, "pepper.png");
+            final Member to = new Member("이브", 123123, "image.png");
+            final Team team = new Team("잠실 네오조", "트랙룸", LocalDateTime.now().plusDays(3), "jamsil.img");
+            final Levellog levellog = Levellog.of(to, team, "스프링을 공부하였습니다.");
+            final String content = "스프링이란?".repeat(255);
+
+            // when & then
+            assertThatThrownBy(() -> InterviewQuestion.of(from, to, levellog, content))
+                    .isInstanceOf(InvalidFieldException.class)
+                    .hasMessageContainingAll("인터뷰 질문은 255자 이하여야합니다.", String.valueOf(content.length()));
+        }
+    }
+
+    @Nested
+    @DisplayName("updateContent 메서드는")
+    class UpdateContentTest {
+
+        @Test
+        @DisplayName("인터뷰 질문 내용을 수정한다.")
+        void success() {
+            // given
+            final Member from = new Member("페퍼", 1111, "pepper.png");
+            final Member to = new Member("이브", 123123, "image.png");
+            final Team team = new Team("잠실 네오조", "트랙룸", LocalDateTime.now().plusDays(3), "jamsil.img");
+            final Levellog levellog = Levellog.of(to, team, "레벨로그 작성 내용");
+            final InterviewQuestion interviewQuestion = InterviewQuestion.of(from, to, levellog, "스프링이란?");
+
+            // when
+            final String updatedContent = "스프링 빈이란?";
+            interviewQuestion.updateContent(updatedContent, from);
+
+            // then
+            assertThat(interviewQuestion.getContent())
+                    .isEqualTo(updatedContent);
```

Comment:
여기는 한줄로 표현해도 되지 않을까용
```suggestion
            assertThat(interviewQuestion.getContent()).isEqualTo(updatedContent);
```

Reply:
객체지향생활체조에 따라 `.`을 한줄에 하나씩 써야하나 했었는데 저희 레벨로그 코드에서 `.isEqualTo()`는 한 줄에 사용하고 있었네용! 반영하였습니당👍